.github/workflows/release.yml

```diff
@@ -0,0 +1,74 @@
+name: Release
```

Comment:
please rename file to `docker-release.yml` (or release-docker)

Reply:
No, because Docker build it's only one job related to the release. It's described in the commit's message:

> Workflow `Release` can be later extended with official site publication, social media announcements etc.

It does not make sense to create separate workflows with the same trigger (SemVer tags), we just need to add another jobs that should be executed on each release.